bundles/org.eclipse.ui.workbench.texteditor/src/org/eclipse/ui/texteditor/AccessibleToolItemBuilder.java

```diff
@@ -0,0 +1,61 @@
+package org.eclipse.ui.texteditor;
+
+import org.eclipse.swt.SWT;
+import org.eclipse.swt.events.SelectionListener;
+import org.eclipse.swt.graphics.Image;
+import org.eclipse.swt.widgets.ToolItem;
+
+/**
+ * Builder for ToolItems for {@link AccessibleToolBars}.
+ *
+ * @since 3.17
+ */
+public class AccessibleToolItemBuilder {
```

Comment:
Reformat the code, I see some indentation problems. Please also reformat other code you added, just in case. Just the **added** code please.

Reply:
I have performed Ctrl+Shift+F on the newly added files again. Note that sometimes the github formatting does not reflect the formatting you get inside of eclipse.

Reply:
Can you please check if https://github.com/eclipse-platform/eclipse.platform.ui/commit/0c0d22bc8f0ea13eb6e1124c89747f7a4f02e266 fixes your issues?

Reply:
It does, thank you